src/app/features/psbt-signer/components/psbt-inputs-outputs-totals/components/psbt-address-receive-totals.tsx

```diff
@@ -8,23 +8,46 @@ import { PsbtAddressTotalItem } from './psbt-address-total-item';
 import { PsbtInscription } from './psbt-inscription';
 
 interface PsbtAddressTotalsProps {
+  showNativeSegwitTotal: boolean;
   showTaprootTotal: boolean;
 }
-export function PsbtAddressReceiveTotals({ showTaprootTotal }: PsbtAddressTotalsProps) {
-  const { accountInscriptionsBeingReceived, addressTaproot, addressTaprootTotal } =
-    usePsbtSignerContext();
+export function PsbtAddressReceiveTotals({
+  showNativeSegwitTotal,
+  showTaprootTotal,
+}: PsbtAddressTotalsProps) {
+  const {
+    accountInscriptionsBeingReceived,
+    addressNativeSegwit,
+    addressNativeSegwitTotal,
+    addressTaproot,
+    addressTaprootTotal,
+  } = usePsbtSignerContext();
   const calculateBitcoinFiatValue = useCalculateBitcoinFiatValue();
 
-  const isReceivingInscriptions = accountInscriptionsBeingReceived?.length;
+  const isReceivingInscriptions = accountInscriptionsBeingReceived?.length > 0;
 
   return (
     <>
+      {!isReceivingInscriptions && showNativeSegwitTotal ? (
+        <PsbtAddressTotalItem
+          hoverLabel={addressNativeSegwit}
+          subtitle={truncateMiddle(addressNativeSegwit)}
+          subValue={i18nFormatCurrency(calculateBitcoinFiatValue(addressNativeSegwitTotal)).replace(
+            '-',
+            ''
+          )}
```

Comment:
Why are we doing this?

Reply:
We subtract inputs from outputs to determine transfer vs receive, but we don't need to show the value displayed as negative if receiving.

Reply:
Maybe we can make a remove minus sign function than repeating?